website/content/docs/api-clients/client-agent.mdx

```diff
@@ -74,14 +73,30 @@ type=target;ids=*;actions=authorize-session
 type=auth-token;ids=*;actions=read:self
 ```
 
+Boundary also requires the following grant string for transparent sessions so that you can resolve the aliases associated with targets:
+
+```
+type=target;actions=list,read
+```
+
```

Comment:
Wait, sorry, where did this requirement come from? I don't think we need this, do we? Was this also a grant missing from the CLI bug? CC @RyanDerr. Also, should this be

```
ids=*;type=target;actions=list,read
```

Reply:
If this is another grant that's usually automatically added, I think we should just have a note about it for older clusters, not imply that all HCPb customers need to do this.

Reply:
Thanks for confirming Ryan, that makes sense to me :)

Reply:
Thanks for the clarification! I removed these lines and included that grant string in the Note that Johan suggested below.